src/data/types.js

```diff
@@ -41,6 +41,7 @@
  * @property {string} commands[].keypresses[].keystroke single human-readable key or key chord press
  * @property {string} [commands[].extraInstruction] human-readable additional instruction to follow
  * @property {string} [commands[].settings] this property only exists on v2 tests
+ * @property {string[]} [commands[].additionalSettings] this property only exists on v2 tests
```

Comment:
This change isn't present in [Howard's patch](https://github.com/w3c/aria-at/pull/1125). Since the file needs to match [the corresponding file in the aria-at repository](https://github.com/w3c/aria-at/blob/bab79e2321ae203e8cfdd464871809247fddd3c3/types/aria-at-file.js), we'll need to resolve the discrepancy before merging.

Reply:
yep, i noticed that too! i have a PR open to Howard's patch here: https://github.com/w3c/aria-at/pull/1128